web/src/classic/components/organisms/Dashboard/hooks.ts

```diff
@@ -192,26 +194,43 @@ export default (workspaceId?: string) => {
         });
         setModalShown(false);
         return;
-      } else {
-        const scene = await createScene({
-          variables: { projectId: project.data.createProject.project.id },
+      }
+
+      const scene = await createScene({
+        variables: { projectId: project.data.createProject.project.id },
+      });
+      if (scene.errors || !scene.data?.createScene?.scene.id) {
+        setNotification({
+          type: "error",
+          text: t("Failed to create project."),
         });
-        if (scene.errors) {
+        setModalShown(false);
+        return;
+      }
+
+      if (data.projectType === "beta") {
+        const story = await createStory({
+          sceneId: scene.data?.createScene?.scene.id,
+          title: "Default",
```

Comment:
Can you i18n the title? 🙏🏻

Reply:
This doesn't seem to be resolved

Reply:
didn't fix yet in that time..:pray:
Fixed in https://github.com/reearth/reearth/pull/592/commits/276e09ef78d3e5a27d40fc80c5f268c06d6ae306